src/actions.ml

```diff
@@ -1623,11 +1627,8 @@ let coq_bug_minimizer_resume_ci_minimization_action ~bot_info ~key ~app_id body
       ; pr_number ] -> (
         message |> String.split ~on:'\n'
         |> function
-        | docker_image
-          :: target
-             :: opam_switch
-                :: failing_urls
-                   :: passing_urls :: base :: head :: bug_file_lines ->
+        | docker_image :: target :: opam_switch :: failing_urls :: passing_urls
+          :: base :: head :: bug_file_lines ->
```

Comment:
OCaml format seems to have picked this up, shall I drop it?

Reply:
Are you using 0.21? If yes, then let it be, and it's just me who needs to update.